libs/full/performance_counters/include/hpx/performance_counters/counters_fwd.hpp

```diff
@@ -28,44 +28,44 @@ namespace hpx { namespace performance_counters {
     inline std::string remove_counter_prefix(std::string const& counter);
 
     ///////////////////////////////////////////////////////////////////////////
-    enum counter_type
+    enum class counter_type
     {
-        // \a counter_text shows a variable-length text string. It does not
+        // \a text counter shows a variable-length text string. It does not
         // deliver calculated values.
         //
         // Formula:  None
         // Average:  None
         // Type:     Text
-        counter_text,
+        text = 0,
```

Comment:
Do we need the explicit `= N`? I don't think we care about particular values here.

Reply:
Let me try it. Thanks for the tip.